src/app/common/settings-buttons.ts

```diff
@@ -475,6 +477,13 @@ export const SettingsButtons: SettingsButtonsType = {
     title: 'BUTTONS.playlistButton',
     toggled: false
   },
+  'randomizeFoldersScreenshots': {
+    description: 'BUTTONS.randFolderScreenDesc',
+    hidden: true,
+    iconName: 'icon-image',
```

Comment:
I wonder if this button would ever be useful in the _buttons ribbon_ 🤔 
I suspect the users would likely toggle it once and never fidget with it after.

I appreciate that you found an icon for the button, but what are your thoughts on removing this property (and the `svg`) - and the button will appear as a plain toggle in settings-only?

Reply:
Ok, yeah i think you are right.
I'll get to it later today.